subprojects.cfg

```diff
@@ -81,4 +81,4 @@ transport:transport-runtime-testing
 
 #firebase-storage:test-app
 #appcheck:firebase-appcheck:test-app
-#firebase-appdistribution:test-app
+firebase-appdistribution:test-app
```

Comment:
I *think* this change should not be committed because of reasons

Reply:
Sorry yes please refresh